src/main/java/edu/kis/vh/nursery/DefaultCountingOutRyhmer.java

```diff
@@ -7,6 +7,8 @@
 
     // klasa wykorzystujac wspolny interjejs moze korzystac teraz z IntArrayStack lub IntLinkedList
 
+    //Wszystkie klasy Ryhmers lamia ta zasade, brak im wspolnego interfejsu.
```

Comment:
Only one rhymer breaks isolation between abstraction and implementation hierarchies.